public/docs/getting_started/client.md

```diff
@@ -1,20 +1,54 @@
+# Choosing an Ethereum client
+
 There are many Ethereum clients to choose from. We recommend using different clients when developing and deploying.
 
-# When Developing
+## Developing locally
+
+### Truffle Develop
+
+We recommend using Truffle Develop, an Ethereum client built directly into Truffle which contains an in-memory blockchain that runs only on your development machine.
+
+It processes transactions instantly instead of waiting for the default block time, so you can test that your code works quickly, and it tells you immediately when your smart contracts run into errors. It also makes a great client for automated testing.
+
+Because Truffle Develop is built into Truffle, no external installation is required. To run Truffle Develop, type the following in a terminal:
```

Comment:
I prefer "into" instead of "in" in the phrase "in a terminal here". Either works however if you have strong opinions.

Reply:
Using "into" is fine there. If it were a preceding clause, I'd prefer "in" (as in, "In a terminal, type the following:").